service/src/main/java/com/codeforcommunity/requester/S3Requester.java

```diff
@@ -273,4 +282,48 @@ public static String uploadHTML(
 
     return String.format("%s/%s/%s", externs.getBucketPublicUrl(), directoryName, name);
   }
+
+  // helper to check whether the given path exists
+  public static boolean pathExists(String path) {
+    Boolean exists = externs.getS3Client().doesObjectExist(externs.getBucketPublic(), path);
```

Comment:
:pick: Can we turn this method into one line? Maybe something like `return externs.getS3Client().doesObjectExist(externs.getBucketPublic(), path);`